### PR TITLE
Create House of NFTs for the wiki

### PR DESCRIPTION
**Pull request by SLRead**
New file for new page on wiki covering NFTs.  I have completed the Art NFTs (since that's what I know) and provided an outline with a few examples of other types of NFTs.  These can be filled out more completely by anyone with knowledge in any of these areas.  (I'll ask on Discord for suggestions...)

Also, fixed the typo reported in introduction.mdx - as per Discord comment.

### CODE REVIEW

**Comment by luxumbra** [comment]
hey @SLRead, sorry, I only just noticed your PR. I'll take a look. Thanks so much for this! :pray: 

Did you get round to asking for suggestions in Discord? Shout if you want any help rallying folk.

File: docs/enter-metagame/how-to-become-a-player.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Ain't no freeloading in MetaGame. Luckily, there are countless ways of contribut
 -   Least you can do is report on your progress, whether its something you're learning or the project you're working on. Just _[#](https://discord.gg/dU2pnEp)_[🌞](https://discord.gg/dU2pnEp)_[-check-in](https://discord.gg/dU2pnEp)._
     
 
-To get inspired on how you could contribute: read through [the roles](https://wiki.metagame.wtf/docs/enter-metagame/roles-in-metagame) or check out the _[ongoing raids](https://wiki.metagame.wtf/docs/enter-metagame/navigation-board)_. Find the role that fits you best, a raid you are most drawn to, or just do whatever you think is valuable & report in _[#](https://discord.gg/JAFX3Ry)_[😊](https://discord.gg/JAFX3Ry)_[-I-did-this](https://discord.gg/JAFX3Ry)_.
+To get inspired on how you could contribute: read through [the roles](https://wiki.metagame.wtf/docs/enter-metagame/roles-in-metagame) or check out the _[ongoing raids](https://wiki.metagame.wtf/docs/enter-metagame/navigation-board)_. Find the role that fits you best, a raid you are most drawn to, or just do whatever you think is valuable & report in _[#](https://discord.gg/JAFX3Ry)_[😊](https://discord.gg/JAFX3Ry)_[-did-a-thing](https://discord.gg/JAFX3Ry)_.
```
Comment (luxumbra) [inline docs/enter-metagame/how-to-become-a-player.md:23]: @SLRead thanks for sorting this but we have a conflict here where you've rightly fixed the did-a-thing channel name but it's already been fixed and merged in a PR from someone else. Sorry about that.

**Review by luxumbra — COMMENTED** [review]
If you pull the latest code from MetaFam:master, you'll pick up the changes and if you get a merge conflict, accept the 'incoming' changes. Then, push your branch again. :pray:

File: docs/enter-metagame/how-to-become-a-player.md (hunk 1 above)
Comment (SLRead) [reply docs/enter-metagame/how-to-become-a-player.md:23]: No worries!  I'll remove the conflict and get this sorted!  Thanks for reviewing!

**Review by hammadj — APPROVED** [review]

**Pull request merged**